fix(migration guide): Update the notes on custom nodes installation path (#1542)
n8n-io/n8n-docs: docs/1-0-migration-checklist.md

### Directory for installing custom nodes

n8n will no longer load custom nodes from its global `node_modules` directory. Instead, you must install (or link) them to `~/.n8n/custom` (or a directory defined by `CUSTOM_EXTENSION_ENV`). Custom nodes that are npm packages will be located in `~/.n8n/nodes`.

> If you have a custom nodes that were linked using `npm link` into the global `node_modules` directory, You'd need to link the again, but into `~/.n8n/nodes` instead.

Reviewer: ```suggestion
If you have custom nodes that were linked using `npm link` into the global `node_modules` directory, you need to link them again, into `~/.n8n/nodes` instead.
```
